content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -364,13 +364,17 @@ are different ways to arrange for pods to work on different things.
 This table shows the required settings for `.spec.parallelism` and `.spec.completions` for each of the patterns.
 Here, `W` is the number of work items.
 
-|                             Pattern                                  | `.spec.completions` |  `.spec.parallelism` |
-| -------------------------------------------------------------------- |:-------------------:|:--------------------:|
-| [Job Template Expansion](/docs/tasks/job/parallel-processing-expansion/)           |          1          |     should be 1      |
-| [Queue with Pod Per Work Item](/docs/tasks/job/coarse-parallel-processing-work-queue/)   |          W          |        any           |
-| [Queue with Variable Pod Count](/docs/tasks/job/fine-parallel-processing-work-queue/)  |          1          |        any           |
-| Single Job with Static Work Assignment                               |          W          |        any           |
-
+|             Pattern                       | `.spec.completions` |  `.spec.parallelism` |
+| ----------------------------------------- |:-------------------:|:--------------------:|
+| [Job Template Expansion]                  |          1          |     should be 1      |
+| [Queue with Pod Per Work Item]            |          W          |        any           |
+| [Queue with Variable Pod Count]           |         null        |        any           |
+| [Indexed Job with Static Work Assignment] |          W          |        any           |
+
+[Job Template Expansion]: /docs/tasks/job/parallel-processing-expansion/
```

Comment:
You might want to move this one to the end of the related lists, as it's not a very usable technique.

Reply:
can you clarify which lists are you referring to?

Reply:
I'm suggesting a new ordering that puts “Job Template Expansion” in _last_ place:
```markdown
[Queue with Pod Per Work Item]: /docs/tasks/job/coarse-parallel-processing-work-queue/
[Queue with Variable Pod Count]: /docs/tasks/job/fine-parallel-processing-work-queue/
[Indexed Job with Static Work Assignment]: /docs/tasks/job/indexed-static-parallel-processing/
[Job Template Expansion]: /docs/tasks/job/parallel-processing-expansion/
```

Reply:
Oh, good one.
Done